### PR TITLE
[DDW-561] Fix crashes when Yubikey is connected

### PR DESCRIPTION
**Pull request by danielmain**
This PR fixes a crash happening when a smart card / yubikey is inserted when Daedalus is running in Windows 10

## Todos

- [x] Patch usb library and recompile for Windows
- [x] Create new nix/windows-usb-libs.zip

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1631713877246000)
- [ ] Test only on Windows all HWs

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by danielmain** [comment]
This is a patch by fixing this Issue https://github.com/tessel/node-usb/issues/386
TODO: watch https://github.com/tessel/node-usb if they fix it, so we do not need to use the patched version anymore.

**Comment by miorsufianiohk** [comment]
@danielmain ,

I have tested build 19071 Windows Mainnet/Testnet against Yubikey + HWs. Looked good. No crashes anymore. Also, tested against Mac/Linux just in case. Also looked good 👍 . 

Before approving, could other QAs help to double check just in case please, thanks. cc: @ManusMcCole @dmitrii-gaico

**Review by dmitrii-gaico — APPROVED** [review]

**Review by miorsufianiohk — APPROVED** [review]
LGTM. Tested agains 19071 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Changelog
 
 - Updated Electron and related packages ([PR 2206](https://github.com/input-output-hk/daedalus/pull/2206))
 - Fixed some Japanese translations for the external currencies ([PR 2667](https://github.com/input-output-hk/daedalus/pull/2667))
+- Fix crashes when Yubikey is connected ([PR 2673](https://github.com/input-output-hk/daedalus/pull/2673))
```
Comment (nikolaglumac) [inline CHANGELOG.md:14]: @danielmain please move this line to the top of "Fixes" section and make sure to use past tense - "Fix crashes..." -> "Fixed crashes..." 🙏

**Review by nikolaglumac — CHANGES_REQUESTED** [review]
Great fix @danielmain!
There is just one tiny comment I have left...

File: CHANGELOG.md (hunk 1 above)
Comment (danielmain) [reply CHANGELOG.md]: done

Comment (nikolaglumac) [reply CHANGELOG.md]: Thanks 😊

**Review by nikolaglumac — APPROVED** [review]
Great fix! 💯

**Pull request merged**